frontend/components/password-change/pug/password-change.pug

```diff
@@ -11,10 +11,22 @@
 						label(ng-if="vm.passwordStrength === undefined") New Password
 						input(
 							name="new"
+							aria-label="password entry"
 							type="password"
 							ng-model='vm.newPassword'
 							ng-required
 							ng-keypress="vm.passwordChange($event)")
+					md-input-container.md-block
+						label(ng-if="vm.registerMessage !== undefined") Confirm Password {{vm.registerMessage}}
+						label(ng-if="vm.registerMessage === undefined") Confirm Password
+						input(
+							name="confirm"
+							aria-label="password confirm"
+							type="password"
+							ng-model='vm.confirmPassword'
+							ng-required
+							ng-blur="vm.confirmPasswordInput()"
+							ng-keypress="vm.confirmPasswordInput()")
```

Comment:
For ng-keypress, can we make it do both confirmPasswordInput() and passwordChange(event)? I think we want it to confirm the password as we type as well as setting the password if someone presses enter.